source/setup.Rmd

```diff
@@ -1,282 +1,277 @@
-- has a connection to the internet,
+- runs Windows 10 Home, MacOS 13 Ventura, or Ubuntu 22.04,
 - uses a 64-bit CPU,
+- has a connection to the internet,
 - uses English as the default language.
 
-### Git
 
-As shown in Chapter \@ref(Getting-started-with-version-control), 
-Git \index{git!installation} is a very useful tool for version controlling your projects, 
-as well as sharing your work with others. Here's how to install Git on 
-the following operating systems: 
-
-**Windows:** To install
-Git on Windows, go to <https://git-scm.com/download/win> and download the Windows 
-version of Git. Once the download has finished, run the installer and accept 
-the default configuration for all pages.
+## Chapter learning objectives
 
-**MacOS:** To install Git on Mac OS, 
-open the terminal ([how-to video](https://youtu.be/5AJbWEWwnbY)) 
-and type the following command:
+By the end of the chapter, readers will be able to do the following:
 
+- Download the worksheets that accompany this book.
+- Install the Docker virtualization engine.
+- Edit and run the worksheets using JupyterLab running inside a Docker container.
+- Install Git, JupyterLab Desktop, and R packages.
+- Edit and run the worksheets using JupyterLab Desktop.
+
+## Obtaining the worksheets for this book
+
+The worksheets containing exercises for this book 
+are online at [https://worksheets.datasciencebook.ca](https://worksheets.datasciencebook.ca).
+The worksheets can be launched directly from that page using the Binder links in the rightmost
+column of the table. This is the easiest way to access the worksheets, but note that you will not 
+be able to save your work and return to it again later.
+In order to save your progress, you will need to download the worksheets to your own computer and 
+work on them locally. You can download the worksheets as a compressed zip file 
+using [the link at the top of the page](https://github.com/UBC-DSCI/data-science-a-first-intro-worksheets/archive/refs/heads/main.zip).
+Once you unzip the downloaded file, you will have a folder containing all of the Jupyter notebook worksheets
+accompanying this book. See Chapter \@ref(getting-started-with-jupyter) for
+instructions on working with Jupyter notebooks.
+
+## Working with Docker
+
+Once you have downloaded the worksheets, you will next need to install and run
+the software required to work on Jupyter notebooks on your own computer. Doing
+this setup manually can be quite tricky, as it involves quite a few different
+software packages, not to mention getting the right versions of
+everything&mdash;the worksheets and autograder tests may not work unless all the versions are
+exactly right! To keep things simple, we instead recommend that you install
+[Docker](https://docker.com). Docker lets you run your Jupyter notebooks inside
+a pre-built *container* that comes with precisely the right versions of 
+all software packages needed run the worksheets that come with this book.
+\index{Docker}
+
```

Comment:
Should we have a note here about what a container is? something like:

>Note: In software, a **container** is a virtualized operating system user space. This allows software to be run in isolation, and not interfere with the software that is installed in your operating system's user space. This means that you can use a container to run a specific version of the R programming language (and the necessary packages) for the book's worksheets even if you have a different version of R installed on your computer in your operating system user space. Or even if you haven't installed R at all!

Reply:
yes!